plugin/src/main/kotlin/org/gradle/github/dependencygraph/internal/DependencyGraphRenderer.kt

```diff
@@ -0,0 +1,14 @@
+package org.gradle.github.dependencygraph.internal
```

Comment:
Shouldn't this be a part of the public API if others are supposed to implement it?